backend/private-graph/graph/schema.graphqls

```diff
@@ -494,13 +494,28 @@ type S3File {
 	key: String
 }
 
-type LogLine {
+type Log {
 	timestamp: Timestamp!
 	severityText: SeverityText!
 	body: String!
 	logAttributes: Map!
 }
 
+type LogEdge {
+	cursor: String!
+	node: Log!
+}
+
+type PageInfo {
+	hasNextPage: Boolean!
+	endCursor: String!
+}
+
+type LogsPayload {
```

Comment:
See the [relay spec](https://relay.dev/graphql/connections.htm) for why things are called what they are called.